src/commands/kv/bucket.rs

```diff
@@ -0,0 +1,103 @@
+extern crate base64;
+
+use crate::commands::kv::delete_bulk::delete_bulk;
+use crate::commands::kv::write_bulk::write_bulk;
+
+use cloudflare::endpoints::workerskv::write_bulk::KeyValuePair;
+
+use walkdir::WalkDir;
+
+use std::ffi::OsString;
+use std::fs::metadata;
+use std::path::Path;
+
+use failure::bail;
+
+use crate::terminal::message;
+
+pub fn upload(namespace_id: &str, filename: &Path) -> Result<(), failure::Error> {
+    let pairs: Result<Vec<KeyValuePair>, failure::Error> = match metadata(filename) {
+        Ok(ref file_type) if file_type.is_dir() => parse_directory(filename),
+        Ok(_file_type) => {
+            // any other file types (namely, symlinks)
```

Comment:
This comment is old! We can replace with
```suggestion
            // any other file types (files, symlinks)
```